debian/control

```diff
@@ -3,7 +3,7 @@ Section: web
 Priority: optional
 Maintainer: Debian letsencrypt <letsencrypt-devel@lists.alioth.debian.org>
 Uploaders: Francois Marier <francois@debian.org>
-Build-Depends: debhelper (>= 9), python-all, dh-python
+Build-Depends: debhelper (>= 9), python-all, dh-python, python-m2crypto, python-zope.component, python-augeas, python-psutil, python-jsonschema, python-requests, python-nose, python-openssl, python-crypto, python-nosexcover, python-tox, python-mock, python-confargparse
```

Comment:
Most of these are necessary for runtime. Is `Build-Depends` sufficient? Shouldn't it go to `Depends` instead?